test/base_results/baseline/annual/results_output.csv

```diff
@@ -251,7 +251,7 @@ project_national-0249.osw,1,0.0,6407.352,0.0,33246.081,0.0,39653.433,0.0,0.0,0.0
 project_testing-0002.osw,0,0.0,3482.268,0.0,25.861,0.0,3508.129,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.232,0.475,0.048,0.0,0.0,0.053,0.135,6.319,3.19,0.0,0.0,0.0,0.0,2.012,1.987,1.605,0.0,0.36,0.219,0.0,1.563,0.797,0.0,0.0,3.004,0.0,0.0,0.0,1.515,2.914,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.113,0.0,0.05,0.0,0.0,0.0,0.0,0.013,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,26.605,26.605,0.0,26.429,26.429,0.0,0.176,0.0,0.0,0.0,62.063,150.058,4439.009,11142.818,0.125,0.109,7.811,0.0,6.671,0.467,2891.81,2826.06,5.811,4.979,0.0,0.0,0.0,0.0,0.0,0.0,333.0,0.0,0.0,333.0,24.8,0.0,333.0,0.0,0.0,0.0,0.0,30.0,0.0,0.0,304.7,304.7,0.0,54.9,15.11,12.75,13.9,1.95,2.19,2.15,0.63,0.59,0.61,16.0,14.09,2.85,2.9,5342.0,4.76
-project_testing-0003.osw,0,0.0,5404.31,0.0,5420.105,900.149,11724.564,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.325,0.0,0.0,0.0,0.0,3.095,0.547,0.0,1.091,0.0,0.264,0.0,0.0,4.835,4.777,2.544,0.0,0.397,0.0,0.0,0.0,0.075,0.0,0.0,7.617,0.0,0.0,-13.153,0.173,5.645,0.0,1.269,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,8.459,0.0,4.164,0.0,24.173,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,5.063,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,61.36,74.512,0.0,19.501,32.653,0.0,36.796,5.063,0.0,0.0,0.0,1192.867,11316.111,32110.262,1.489,19.768,18.875,0.0,9.19,1.308,2839.88,2850.97,6.362,26.409,0.0,11.0,30.0,0.0,0.0,0.0,1675.0,0.0,558.3,0.0,0.0,76.0,624.2,0.0,0.0,19.03,0.0,80.0,0.0,0.0,1322.1,1322.1,535.4,188.6,12.67,17.1,15.65,1.96,1.93,1.99,0.66,0.64,0.63,16.09,16.18,2.62,2.2,8200.0,2.85
+project_testing-0003.osw,0,0.0,5367.141,0.0,5282.852,900.149,11550.142,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.325,0.0,0.0,0.0,0.0,2.954,0.547,0.0,1.091,0.0,0.259,0.0,0.0,4.835,4.777,2.542,0.0,0.397,0.0,0.0,0.0,0.075,0.0,0.0,7.617,0.0,0.0,-13.153,0.173,5.645,0.0,1.269,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,8.459,0.0,4.164,0.0,23.242,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,5.063,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,60.281,73.433,0.0,19.353,32.506,0.0,35.865,5.063,0.0,0.0,0.0,1192.872,11311.107,32096.061,1.487,19.072,18.875,0.0,9.192,1.305,2600.17,2850.84,6.483,25.876,0.0,11.0,30.0,0.0,0.0,0.0,1675.0,0.0,558.3,0.0,0.0,76.0,624.2,0.0,0.0,18.59,0.0,80.0,0.0,0.0,1196.5,1196.5,535.4,176.0,12.6,17.08,15.75,1.95,1.93,1.98,0.66,0.64,0.63,16.18,16.18,2.59,2.2,8200.0,2.85
```

Comment:
Can you confirm that these three testing samples changed because they are SFA with cathedral ceilings?

Reply:
Confirmed, each of those samples (0003, 0021, 0069) are end unit, SFA + cathedral ceilings, and the space conditioning energy use changes as expected.